include/independent_set/CographIndependentSet.hpp

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include "recognition/CographAlg.hpp"
+
+namespace Koala {
+    class CographIndependentSet {
```

Comment:
Please inherit from the IndependentSet class (as you did for VertexColoring) + please also provide the respective abstract classes for other algorithms like Pathwidth